plone/app/querystring/tests/testQueryParser.py

```diff
@@ -127,398 +138,345 @@ def getAuthenticatedMember(self):
-            'v': 'Welcome to Plone',
+            "i": "Title",
+            "o": "plone.app.querystring.operation.string.is",
+            "v": "Welcome to Plone",
         }
         parsed = queryparser.parseFormquery(
-            MockSite(), [data, ],
-            sort_on='sortable_title',
-            sort_order='reverse')
+            MockSite(),
+            [
+                data,
+            ],
+            sort_on="sortable_title",
+            sort_order="reverse",
+        )
         self.assertEqual(
-            parsed, {'Title': {'query': 'Welcome to Plone'},
-                     'sort_on': 'sortable_title',
-                     'sort_order': 'reverse'})
+            parsed,
+            {
+                "Title": {"query": "Welcome to Plone"},
+                "sort_on": "sortable_title",
+                "sort_order": "reverse",
+            },
+        )
 
     def test_sort_on_unknown(self):
         data = {
-            'i': 'Title',
-            'o': 'plone.app.querystring.operation.string.is',
-            'v': 'Welcome to Plone',
+            "i": "Title",
+            "o": "plone.app.querystring.operation.string.is",
+            "v": "Welcome to Plone",
         }
         parsed = queryparser.parseFormquery(
-            MockSite(), [data, ],
-            sort_on='unknown',
-            sort_order='reverse')
-        self.assertEqual(
-            parsed, {'Title': {'query': 'Welcome to Plone'}})
+            MockSite(),
+            [
+                data,
+            ],
+            sort_on="unknown",
+            sort_order="reverse",
+        )
+        self.assertEqual(parsed, {"Title": {"query": "Welcome to Plone"}})
 
     def test_path_explicit(self):
         data = {
-            'i': 'path',
-            'o': 'plone.app.querystring.operation.string.path',
-            'v': '/foo',
+            "i": "path",
+            "o": "plone.app.querystring.operation.string.path",
+            "v": "/foo",
         }
-        parsed = queryparser.parseFormquery(MockSite(), [data, ])
-        self.assertEqual(
-            parsed, {'path': {'query': ['/%s/foo' % MOCK_SITE_ID]}})
+        parsed = queryparser.parseFormquery(
+            MockSite(),
+            [
+                data,
+            ],
+        )
+        self.assertEqual(parsed, {"path": {"query": ["/%s/foo" % MOCK_SITE_ID]}})
 
     def test_path_computed(self):
         data = {
-            'i': 'path',
-            'o': 'plone.app.querystring.operation.string.path',
-            'v': '00000000000000001',
+            "i": "path",
+            "o": "plone.app.querystring.operation.string.path",
+            "v": "00000000000000001",
         }
-        parsed = queryparser.parseFormquery(MockSite(), [data, ])
-        self.assertEqual(
-            parsed, {'path': {'query': ['/%s/foo' % MOCK_SITE_ID]}})
+        parsed = queryparser.parseFormquery(
+            MockSite(),
+            [
+                data,
+            ],
```

Comment:
_ditto_

there are quite a few others, it's pointless to keep adding comments for each of them 😅